arangod/Aql/BindParameters.cpp

```diff
@@ -65,28 +66,42 @@ uint64_t BindParameters::hash() const {
   return _builder->slice().hash();
 }
 
-/// @brief mark a bind parameter as "used", and return its value.
+/// @brief return a bind parameter value by name
 /// will return VPackSlice::noneSlice() if the bind parameter does not exist!
-VPackSlice BindParameters::markUsed(std::string const& name) noexcept {
+std::pair<VPackSlice, AstNode*> BindParameters::get(
```

Comment:
It no longer returns only a value, so the comment should be updated accordingly.

Reply:
Done